.github/workflows/tests.yml

```diff
@@ -282,7 +284,7 @@ jobs:
         uses: actions/upload-artifact@main
         with:
           name: logs-${{ matrix.runner }}
-          path: ${{matrix.workdir || github.workspace}}/bottles/logs
+          path: ${{ runner.os == 'Linux' && needs.setup_tests.outputs.logs-dir || format('{0}/bottles/logs', matrix.workdir || github.workspace) }}
```

Comment:
Any way se can set `core.setOutput('logs-dir', ` above for all cases so we don't need as much in here?

Reply:
Thought about it but not really. Proper workdir or workspace variables are only accessible inside this job so we can't set the output in the previous one. Maybe we could set it in another step in this job before uploading step but I don't think it will be any more readable. There might be a better way for the implementation but I can't think of one currently. Plus all those "hacks" should go away once we have all of our runners be ephemeral, dunno how close we are to this tho.

Reply:
Ok no worries.